### ISSUE TITLE
[css-grid-3] The initial value of `item-tolerance` should be 0

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#10882]
https://drafts.csswg.org/css-grid-3/#masonry-slack says that the initial is `masonry-slack: 1em`.

I think that the correct `masonry-slack` slack value is very dependent on the contents of the masonry, so `1em` may not work well in some cases, and will probably just cause unnecessary confusion to authors.

I think it's better to default to 0, and then let authors choose what works best for them.

**Comment by fantasai** [comment]
I disagree, because of I think the default should be something that generally works well: and I suspect there are very, very few use cases where a value less than `1em` would be appropriate.

It's also easier to understand that there's a value you can tweak here when it isn't zero: the behavior is already there, so you go looking for the knob rather than assuming there isn't one.

`1em` isn't particularly arbitrary; we use it for `column-gap`. The one change that might make sense is switching it to `1lh` since it's a block-axis adjustment.

**Comment by tabatkins** [comment]
Yeah, a 1px difference between two columns should *definitely not* cause the masonry items to be misordered by default; that would just be a bad default behavior. We can discuss exactly what value the default should be, but it should definitely be something non-zero but fairly small, *approximately* `1em`.

**Comment by Loirooriol** [comment]
> a 1px difference between two columns should definitely not cause the masonry items to be misordered by default

I have tested desandro's Masonry and Macy.js, and both definitely do that by default. I didn't even see an option to change this, so it doesn't seem something authors expect or are asking for (unlike #10883 btw).

While I agree that using some positive value is desirable in most cases, I'm not convinced that the confusion that it may cause if we set `1em` by default is a net gain.

> we use it for `column-gap`

Yeah I guess that's a good point, though.

> I really like the idea of `1lh` as the default.

I don't see `1lh` as a particularly better choice than `1em`, it seems to me that masonry is typically used for image galleries where the height of a text line isn't much relevant.

**Comment by fantasai** [comment]
Thinking about it more, `1lh` makes sense in the block axis, but not so much in the inline axis, so I think we should just keep it as `1em`.

> While I agree that using some positive value is desirable in most cases, I'm not convinced that the confusion that it may cause if we set 1em by default is a net gain.

The positioning of items when slack is zero is itself pretty surprising. And worse for accessibility. So strongly believe we should have a noticeable default value.

**Comment by astearns** [comment]
@alisonmaher you had added this to the F2F agenda, but we didn’t get to it. The last time we discussed this it was inconclusive. Let’s see if we can bring it back to the agenda with more information than we currently have.

Comments welcome!

**Comment by astearns** [comment]
@fantasai I explicitly asked for more comments here before bringing this back for another inconclusive discussion. I’m taking the agenda tag off until I see evidence we can make progress.

**Comment by fantasai** [comment]
Diagram drawn during the call: 

<img width="702" height="366" alt="Image" src="https://github.com/user-attachments/assets/84c8fd31-7202-4ca5-9d41-d9ac8beef65a" />

**Comment by fantasai** [comment]
Marking Needs Edits so we can add a diagram. :)
